src/web/modules/stripe/basic/confirm-create-test-account.njk

```diff
@@ -0,0 +1,42 @@
+{% extends "layout/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block main %}
+  {% if systemLinkService %}
+    <div class="govuk-body">
+      <a class="govuk-link govuk-link--no-visited-state" href="/services/{{ systemLinkService }}">View service</a>
+    </div>
+  {% endif %}
+
+  <h1 class="govuk-heading-m">Create test Stripe account</h1>
+
+  {{ govukTable({
+    firstCellIsHeader: true,
+    rows: [
+      [
+        { text: "Service name" },
+        { text: serviceName }
+      ]
+    ]
+    })
+  }}
+
+  <form method="POST" action="/stripe/basic/create-test-account">
+    <p class="govuk-body">You are about to create Stripe account for the service. This will create:</p>
+    <ul class="govuk-list govuk-list--number">
+      <li>Stripe test connect account through the Stripe API</li>
+      <li>GOV.UK Pay stripe test gateway account, which will be available for service immediately on selfservice Myservices page</li>
```

Comment:
```suggestion
      <li>GOV.UK Pay test gateway account setting the provider to Stripe, this will be available for service immediately on the admin tool "My services" page</li>
```